src/app/child-window-handler.ts

```diff
@@ -195,6 +195,7 @@ export const handleChildWindow = (webContents: WebContents): void => {
         } else {
             if (newWinUrl && newWinUrl.length > 2083) {
                 logger.info(`child-window-handler: new window url length is greater than 2083, not performing any action!`);
+                event.preventDefault();
```

Comment:
we can refactor this! Just move `event.preventDefault()` to the top of the `else` block and remove the redundant one.